source/shared/core_sqlsrv.h

```diff
@@ -1099,24 +1103,28 @@ const char SERVER[] = "Server";
 
 enum SQLSRV_CONN_OPTIONS {
    
-   SQLSRV_CONN_OPTION_INVALID,
-   SQLSRV_CONN_OPTION_APP,
-   SQLSRV_CONN_OPTION_CHARACTERSET,
-   SQLSRV_CONN_OPTION_CONN_POOLING,
-   SQLSRV_CONN_OPTION_DATABASE,
-   SQLSRV_CONN_OPTION_ENCRYPT,
-   SQLSRV_CONN_OPTION_FAILOVER_PARTNER,
-   SQLSRV_CONN_OPTION_LOGIN_TIMEOUT,
-   SQLSRV_CONN_OPTION_MARS,
-   SQLSRV_CONN_OPTION_QUOTED_ID,
-   SQLSRV_CONN_OPTION_TRACE_FILE,
-   SQLSRV_CONN_OPTION_TRACE_ON,
-   SQLSRV_CONN_OPTION_TRANS_ISOLATION,
-   SQLSRV_CONN_OPTION_TRUST_SERVER_CERT,
-   SQLSRV_CONN_OPTION_WSID,
-   SQLSRV_CONN_OPTION_ATTACHDBFILENAME,
-   SQLSRV_CONN_OPTION_APPLICATION_INTENT,
-   SQLSRV_CONN_OPTION_MULTI_SUBNET_FAILOVER,
```

Comment:
But most of this file has 4 space indentations - this is one of only two places where the indentation was 3 spaces. Actually I think I should fix the indentation in the other place too (the SQLSRV_STMT_OPTIONS enum at line 1062).